addons/zylann.editor_debugger/dock.gd

```diff
@@ -375,6 +380,14 @@ func _on_popup_menu_id_pressed(id: int) -> void:
 			var node_types_str := "%s"%[node_types]
 			DisplayServer.clipboard_set(node_types_str)
 			print("Copied to clipboard: %s"%[node_types_str])
+		
+		POPUP_ACTIONS.COPY_NODE_CHILD_PATH_INDICES:
+			var node_view := _tree_view.get_selected()
+			var node := _get_node_from_view(node_view)
+			var index_path = get_node_index_path(node)
+			DisplayServer.clipboard_set(path_to_get_child_string(index_path))
+			print("Copied to clipboard: %s"%[path_to_get_child_string(index_path)])
```

Comment:
I'm not sure printing is the right thing to do here, the console might not even be open. Can't it just be the string copied to clipboard in the first place, like you did originally?

Reply:
The original had printing, as do the other instances of "copy node". Would you like me to comment out or remove all prints in the other PR or just combine it all here?

Reply:
I'm mainly wondering what you really want in the clipboard, because prints are not meant to be important. Indeed they might be removed, but those at the moment may be present to acknowledge that something was copied, and that something should be the same as what was put in the clipboard, which is not the case in what you changed.

Reply:
Ah nevermind, they are the same actually. I misread. Could you put it in a variable though, so that it's not repeated twice?

Reply:
Done